packages/teams-js/src/public/app/app.ts

```diff
@@ -27,6 +27,8 @@ import {
 import { version } from '../version';
 import * as lifecycle from './lifecycle';
 
+prefetchOriginsFromCDN();
```

Comment:
Now that this is called in the global namespace this file has a side effect that it didn't used to have (loading the file in memory now automatically makes a call to the CDN). Can you verify with Noah that we don't need to mark this file in any particular way to prevent it from being treeshaken out?

Reply:
Good call, I'll follow up with Noah on this and if there's any issue I'll move this call into a separate funciton.

Reply:
Followed up with Noah, this will in fact introduce an unintended side effect that would make `app.ts` not treeshake-able. I'll be pushing out a change that will mitigate this or lessen the impact of the side effect.

Reply:
Hmmm, that's a good question. I'll think about it (and maybe ask @noahdarveau-MSFT). In general, any function calls in the global namespace are "side effects" if they cause some state somewhere to change because those functions will just get called assuming the entire module doesn't get tree-shaken out and removed. Off the top of my head I'm not sure there's a way to "detect" that.

Reply:
Okay, I am pretty sure that we could make a linter rule to detect global function calls that would show a warning saying something like "This file should probably be added to the sideEffects array in package.json. Once you've done that (or verified that you don't need to) please suppress this warning."